controller/transfer/record_transfer_manager.py

```diff
@@ -81,6 +107,38 @@ def import_records_and_rlas(
             )
 
 
+def force_data_type_for_attributes(
```

Comment:
is there a specific reason why this function is public and the one below is private when both are used in the same file?